doc/ref_model/chapters/chapter07.md

```diff
@@ -171,6 +172,24 @@ The platform supports the workload, and in effect controls access to the workloa
 * Front-end network security at the application level will be the responsibility of the workload, however the platform must ensure the isolation and integrity of tenant connectivity to front-end networks.
 * The front-end network may provide (Distributed Denial Of Service) DDOS support.
 
+<a name="7.4.4"></a>
+### 7.4.4 Infrastructure as a Code security
+Infrastructure as a Code (IaaC) is used for the declarative management of cloud infrastructure resources.  In order to dynamically address user requirements, release features incrementally, and deliver at a faster pace, DevOps teams utilize best practices including continuous integration and continuous delivery. To effectively integrate information security controls into these processes, with the aim of providing timely and meaningful feedback, automated security capabilities will be of critical value.
+
+Because of an extremely high complexity of modern telco cloud infrastructure, even very small IaaC code changes may lead to disproportionate and sometime disastrous downstream impacts. Therefore, integration of security testing into the IaaC software development pipeline requires security activities to be automated and bridged with the native DevOps tools and procedures. 
```

Comment:
Typo and minor suggested change
```suggestion
Because of the extremely high complexity of modern telco cloud infrastructure, even minor IaaC code changes may lead to disproportionate and sometime disastrous downstream impacts. Therefore, integration of security testing into the IaaC software development pipeline requires security activities to be automated and bridged with the native DevOps tools and procedures. 
```